### DISCUSSION TITLE
Firebase for Windows platform

### DISCUSSION THREAD

**Discussion by haspdev** [firebase/flutterfire#6233] [Feature request] [answered]
I have the android application which uses _Firebase Storage, Firebase Auth, Firestore & Firebase database_, but my client wants the windows & mac software for business. Flutter 2 is added support for windows. but  **Firebase lib's not supporting the Windows** platform.  when **firebase** will add **support for the Windows platform**?. 
I request to all the Firebase developers please _add Firebase support for the Windows platform ASAP._ 
Thanks

**Comment by worldwidee** [comment]
@haspdev same issue on me. Someone should at least tell us if this will happen

**Comment by barbalex** [comment, marked as answer]
Duplicate of https://github.com/FirebaseExtended/flutterfire/discussions/5557
